spec/plans/results.fmf

```diff
@@ -0,0 +1,118 @@
+summary: Define format of on-disk storage of results
+title: Results format
+order: 90
+
+description: |
+    The following text defines a YAML file structure tmt uses for storing
+    results. tmt itself will use it when saving results of ``execute`` step,
+    and custom test result are required to follow it when creating their
+    ``results.yaml`` file.
+
+    Results are saved as a YAML file, containing a single list of mappings,
+    one mapping describing a single test result.
+
+    .. code-block::
+
+       # String, name of the test.
+       name: /test/one
+
+       # String, outcome of the test execution.
```

Comment:
Hm, is there a way how to have the  the possible outcome values specified outside of this code-block? There is `ref` used inside and it is not processed. It also looks like an example, not the format specification.

Reply:
Hopefully improved in a8424505.

Reply:
Thanks.